manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+# Deploy code to production
+
+Transfer Digital Records uses the practice of [Continuous Integration]: developers merge their work into the a repo's
+main branch regularly. The repo is configured to only allow changes to be merged if the checks pass, such as unit tests,
+code linters and security checks. These checks are currently run on a [Jenkins instance][tdr-jenkins-intg].
+
+We also perform [Continous Delivery]. The main branch of any repo is always deployable, and can be deployed to
+production multiple times a day if necessary. The [end-to-end tests] which run on integration and staging help us catch
+regression before code is deployed to production. Deploying frequent changes to production means that each deployment is
+small, which makes deployments less risky and easier to fix.
+
+TDR is made up of [multiple services][tdr-architecture], each of which are deployed separately. Infrastructure is also
+defined in Terraform repos, and is deployed to each environment in the same way way.
+
+## When to deploy
+
+You should deploy to Staging and Production as soon as possible after merging the change and checking it on Integration.
+This means that you can deploy your own change. If you leave your change undeployed and someone else deploys it at the
+same time as theirs, and failures it will be harder for them to debug.
+
+We have agreed to some rules about when it is OK to deploy to Production:
+
+- Only when you have time to monitor the deployment. Decide how long you need based on the size and complexity of the
+  change.
+- Only when there are other people around. We are a small team with flexible working hours, so there are no restrictions
```

Comment:
Might be worth adding a points about the e2e tests passing in Intg and Staging before deploying to Production, and not leaving broken deployments overnight if possible?

Reply:
It's a bit pedantic but maybe instead of "other people", it should be more specific like "other developers" or "other developers with permission to deploy to Prod"?

Reply:
@TomJKing I've added an extra bullet point about that, thanks.

@techncl I've made it "other TDR developers" so it's less ambiguous.